src/uu/pkill/src/pkill.rs

```diff
@@ -109,57 +109,13 @@ pub fn uu_app() -> Command {
         .about(ABOUT)
         .override_usage(format_usage(USAGE))
         .args_override_self(true)
-        .group(ArgGroup::new("oldest_newest").args(["oldest", "newest", "inverse"]))
         .args([
             // arg!(-<sig>                    "signal to send (either number or name)"),
-            arg!(-H --"require-handler"    "match only if signal handler is present"),
-            // arg!(-q --queue <value>        "integer value to be sent with the signal"),
+            arg!(-q --queue <value>        "integer value to be sent with the signal"),
```

Comment:
Did you enable this unimplemented arg on purpose?

Reply:
No, rebase mistake... fixed